conjure-java-core/src/main/java/com/palantir/conjure/java/types/UnionGenerator.java

```diff
@@ -68,7 +68,7 @@
     private static final String VALUE_FIELD_NAME = "value";
     private static final String UNKNOWN_WRAPPER_CLASS_NAME = "UnknownWrapper";
     private static final String VISIT_UNKNOWN_METHOD_NAME = "visitUnknown";
-    private static final String COMPLETED = "completed";
+    private static final String COMPLETED = "completed_";
```

Comment:
code change

Reply:
Thanks @carterkozak this fixes my case. Resulting generated class name will be `Completed_StageVisitorBuilder`